### DISCUSSION TITLE
Is there a way to secure the /uploads folder so that files can't be accessed without the "share" feature?

### DISCUSSION THREAD

**Discussion by Julianoe** [LycheeOrg/Lychee#1816] [Q&A] [answered]
I'm sorry if that does not fit anywhere.
I'm trying to understand something. I've been using Lychee for several years but I did not follow the dev recently.
I noticed that I can access original files without being shared a link or logged in. Is that normal ? If not is there a way to prevent it and how?

I'm on the latest version btw.

**Comment by d7415** [comment, marked as answer]
There's an overview here: https://lycheeorg.github.io/docs/faq_general.html#how-is-the-upload-folder-protected
